components/zoom_admin/actions/add-meeting-registrant/add-meeting-registrant.mjs

```diff
@@ -0,0 +1,198 @@
+import zoomLangs from "../../zoom_languages.mjs";
+import get from "lodash/get.js";
+import isArray from "lodash/isArray.js";
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Add meeting registrant",
+  description: "Register a participant for a meeting. [See the docs here](https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingregistrantcreate)",
+  key: "zoom_admin-action-add-meeting-registrant",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    zoomAdmin,
+    meeting: {
+      propDefinition: [
+        zoomAdmin,
+        "meeting",
+      ],
+    },
+    occurrenceId: {
+      propDefinition: [
+        zoomAdmin,
+        "occurrenceId",
+        ({ meeting }) => ({
+          meeting,
+        }),
+      ],
+      type: "string[]",
+      description: "The [meeting occurrence ID](https://support.zoom.us/hc/en-us/articles/214973206-Scheduling-Recurring-Meetings).",
+    },
+    email: {
+      type: "string",
+      label: "Email",
+      description: "A valid email address of the registrant",
+    },
+    firstName: {
+      type: "string",
+      label: "First Name",
+      description: "Registrant's first name",
+    },
+    lastName: {
+      type: "string",
+      label: "Last Name",
+      description: "Registrant's last name",
+      optional: true,
+    },
+    address: {
+      type: "string",
+      label: "Address",
+      description: "Registrant's address",
+      optional: true,
+    },
+    city: {
+      type: "string",
+      label: "City",
+      description: "Registrant's city",
+      optional: true,
+    },
+    country: {
+      propDefinition: [
+        zoomAdmin,
+        "country",
+      ],
+    },
+    zip: {
+      type: "string",
+      label: "ZIP Code",
+      description: "Registrant's Zip/Postal Code.",
+      optional: true,
+    },
+    state: {
+      type: "string",
+      label: "State",
+      description: "Registrant's State/Province.",
+      optional: true,
+    },
+    phone: {
+      type: "string",
+      label: "Phone",
+      description: "Registrant's Phone number.",
+      optional: true,
+    },
+    industry: {
+      type: "string",
+      label: "Industry",
+      description: "Registrant's Industry.",
+      optional: true,
+    },
+    org: {
+      type: "string",
+      label: "Organization",
+      description: "Registrant's Organization.",
+      optional: true,
+    },
+    jobTitle: {
+      type: "string",
+      label: "Job Title",
+      description: "Registrant's Job Title.",
+      optional: true,
+    },
+    purchasingTimeFrame: {
+      type: "string",
+      label: "Purchasing Time Frame",
+      description: "This field can be included to gauge interest of webinar attendees towards buying your product or service.",
+      optional: true,
+      options: [
+        "Within a month",
+        "1-3 months",
+        "4-6 months",
+        "More than 6 months",
+        "No timeframe",
+      ],
```

Comment:
I would set all these variables as constants in a `constants.mjs` file, same goes for the rest for options are there